datajoint/fetch.py

```diff
@@ -37,9 +38,15 @@ def _initialize_behavior(self):
     @property
     def squeeze(self):
         """
+        DEPRECATED
+
         Changes the state of the fetch object to squeeze the returned values as much as possible.
         :return: a copy of the fetch object
         """
+
+        warnings.warn('Use of `squeeze` on `fetch` object is deprecated. Please use `squeeze=True` keyword arguments '
+                      'in the call to `fetch`/`keys` instead', stacklevel=2)
+
         ret = self.copy()
```

Comment:
Perhaps we should deprecate `FetchBase.copy` too.  It seems to be needed only for the old stateful Fetch.